gravitee-apim-e2e/api-test/src/apis/settings/mapi-v1/notification-setting.spec.ts

```diff
@@ -33,7 +33,7 @@ describe('Notification settings tests', () => {
 
   describe('Create', () => {
     it('should create a notification setting', async () => {
-      const users = await managementUserResourceAsAdminUser.getAllUsers({ orgId, envId });
+      const users = await managementUserResourceAsAdminUser.getAllUsers({ orgId, envId, size: 200 });
```

Comment:
Hmm, maybe I should have filtered by name if possible. I didn't check if its doable 🙈